package/linux/application.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=0.1.0
+Encoding=UTF-8
+Name=_GCS
+Exec=_gcs
+TryExec=_gcs
+Comment=Configure, Tune, Diagnose, Track, & Upgrade FW for _ORG solutions
+Terminal=false
+Categories=Utility;Robotics;Engineering;
+Icon=_org
+Type=Application
```

Comment:
I have removed the underscores. I put them in originally to avoid accidentally changing strings already substituted from the earlier sed commands. Anyway, this is not an issue. I have also changed the filename.
One query - the icon name is changed, but the icon is still openpilot.png?